symphony/bdk/core/config/model/bdk_server_config.py

```diff
@@ -0,0 +1,78 @@
+class BdkServerConfig:
+    _DEFAULT_SCHEME: str = "https"
+    _DEFAULT_HTTPS_PORT: int = 443
+
+    def __init__(self, scheme=None, port=None, host=None, context=None, **kwargs):
+        self._scheme = scheme if scheme is not None else self._DEFAULT_SCHEME
```

Comment:
You can put _DEFAULT_SCHEME and _DEFAULT_HTTPS_PORT directly as parameter default values.

Reply:
I can't do that because we are passing arguments using `dict.get()` that is passing `None` as an argument which overrides the default parameters.
```python
>>> BdkConfig
 config = {}
super().__init__(scheme=config.get("scheme"))

#Which overrides the default value in 
>>> BdkServerConfig 
def __init__(self, scheme=None)
```